BoxPreviewSDKSampleApp/BoxPreviewSDKSampleApp.xcodeproj/project.pbxproj

```diff
@@ -432,14 +432,15 @@
 					"$(PROJECT_DIR)/Carthage/Build/iOS",
 				);
 				INFOPLIST_FILE = BoxPreviewSDKSampleApp/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.1;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.box.ios.sampleapp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 3.0;
+				VALID_ARCHS = "arm64 arm64e";
```

Comment:
Are you sure this is needed?

Reply:
I removed arm7 and it seems to Xcode's writes the entry.

Reply:
What I meant was, do we need to remove armv7, or is that implicit in the deployment target?